app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -161,7 +192,7 @@ export function CollectionsPage({ data, params, renderCollections }) {
               onChange: (e) => handleSearchChange(e),
             }}
             onSubmit={handleSearchSubmit}
-            labelText="Search Collections by title"
+            labelText="Search collections by title"
```

Comment:
Just noticing now that this and the labelText on line 188 both are aria labels. They're set for different elements but I wonder if it's redundant (but guess not since it hasn't already been flagged).